ckanext/recombinant/templates/datastore/dictionary.html

```diff
@@ -0,0 +1,51 @@
+{% ckan_extends %}
+
+{% import 'macros/form.html' as form %}
+
+{% block dictionary_form %}
+  {% set chromo = h.recombinant_published_resource_chromo(res.id) %}
+  {% set chromo_keyed = {} %}
+  {% if chromo %}
+    {% for recombinant_field in chromo.fields %}
+      {% do chromo_keyed.update({recombinant_field.datastore_id: recombinant_field}) %}
+    {% endfor %}
+  {% endif %}
+  {% for field in fields %}
+    {% set position = loop.index %}
+    {% if field.id in chromo_keyed %}
+      {% set _type = chromo_keyed[field.id].get('datastore_type', '') %}
+      {% if _type == '_text' %}
+        {% set _type = 'text[]' %}
+      {% endif %}
+      <h3>{{ _( "Field {num}.").format(num=position) }} {{ field.id }} ({{ _type }})</h3>
+      {{ form.input('info__' ~ position ~ '__type_override',
+        label=_('Type Override'), id='info__' ~ position ~ '__type_override',
+        value='', placeholder=_type, attrs={'class': 'form-control', 'readonly': true, 'style': 'pointer-events: none;', 'tabindex': '-1'}) }}
+
+      {{ form.input('info__' ~ position ~ '__label_en',
+        label=_('English Label'), id='field-f' ~ position ~ 'label_en',
+        value='', placeholder=chromo_keyed[field.id].get('label', {}).get('en', ''), classes=['control-full'],
+        attrs={'class': 'form-control', 'readonly': true, 'style': 'pointer-events: none;', 'tabindex': '-1'}) }}
+
+      {{ form.input('info__' ~ position ~ '__label_fr',
+        label=_('French Label'), id='field-f' ~ position ~ 'label_fr',
+        value='', placeholder=chromo_keyed[field.id].get('label', {}).get('fr', ''), classes=['control-full'],
+        attrs={'class': 'form-control', 'readonly': true, 'style': 'pointer-events: none;', 'tabindex': '-1'}) }}
+
+      {{ form.textarea('info__' ~ position ~ '__notes_en',
+        label=_('English Description'), id='field-d' ~ position ~ 'notes_en',
+        value='', placeholder=chromo_keyed[field.id].get('description', {}).get('en', ''),
+        attrs={'class': 'form-control', 'readonly': true, 'style': 'pointer-events: none;', 'tabindex': '-1'}) }}
+
+      {{ form.textarea('info__' ~ position ~ '__notes_fr',
+        label=_('French Description'), id='field-d' ~ position ~ 'notes_fr',
+        value='', placeholder=chromo_keyed[field.id].get('description', {}).get('fr', ''),
+        attrs={'class': 'form-control', 'readonly': true, 'style': 'pointer-events: none;', 'tabindex': '-1'}) }}
+    {% else %}
+      {% snippet "datastore/snippets/dictionary_form.html",
+        field=field, position=position, res=res,
+        data=data.get('fields', [{}] * loop.length)[loop.index0],
+        errors=errors.get('fields', [{}] * loop.length)[loop.index0] %}
+    {% endif %}
```

Comment:
we're still mixing the registry data dictionary info with information from the yaml schema?